### ISSUE TITLE
Ablation

### ISSUE THREAD

**Issue by SijieLuo** [cqylunlun/GLASS#55]
作者您好，请问如何进行模块消融实验？如何实现去掉局部异常合成，或者全局异常合成，谢谢！

**Comment by cqylunlun** [comment]
Hello，需要在训练时更改损失函数的构成。参考[`loss = bce_loss + focal_loss`](https://github.com/cqylunlun/GLASS/blob/cd80bcec67e3576e02c52589b4cc548b5266ad10/glass.py#L420) —— 如果只训练GAS支路，则修改为`loss = bce_loss`；如果只训练LAS支路，则修改为`loss = focal_loss`。

**Comment by SijieLuo** [comment]
好的，谢谢！
